VirtoCommerce.Storefront/Startup.cs

```diff
@@ -383,6 +383,8 @@ public void ConfigureServices(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
+            app.UseForwardedHeaders();
```

Comment:
We already have this extension method
![image](https://user-images.githubusercontent.com/15198683/108997126-eb0c4d00-76a7-11eb-9625-4b66aeea00e4.png)

Reply:
Fixed